bundles/extensions/models/api/src/main/java/org/apache/sling/models/factory/package-info.java

```diff
@@ -14,7 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@Version("1.1.0")
+// TODO check: the baseline plugin forced me to upgrade this to 2.0.0
```

Comment:
This is definitely wrong, probably you have a custom version of Sling Models in the repository you compare with (which is detected as latest version by Maven).